packages/teleterm/src/services/grpcCredentials/types.ts

```diff
@@ -1,6 +1,9 @@
-// `Client` and `TshServer` file names are also used on the tshd side
+// Each process creates its own key pair. The public key is saved to disk under the specified
+// filename, the private key stays in the memory.
+//
+// `Renderer` and `Tshd` file names are also used on the tshd side.
 export enum GrpcCertName {
-  Client = 'client.crt',
```

Comment:
Now that we're going to have both a gRPC client and a gRPC server in the renderer process, the name "client" stopped being sufficient. I refactored this so that it closer reflects what we want to achieve with gRPC certs – each process owns its own key pair, keeps the private key in memory and the public key is saved to the file.